Tools/Plotfile/faverage.cpp

```diff
@@ -0,0 +1,285 @@
+//
+// Laterally average a variable with optional density weighting (Favre average)
+
+#include <iostream>
+// #include <stringstream>
+#include <regex>
+#include <string>
+#include <AMReX_PlotFileUtil.H>
+#include <AMReX_MultiFabUtil.H>
+#include <AMReX_ParallelDescriptor.H>
+
+using namespace amrex;
+
+std::string inputs_name = "";
+
+void PrintHelp ();
+
+
+int main(int argc, char* argv[])
+{
+
+    amrex::Initialize(argc, argv, false);
+
```

Comment:
Let's put everything between `amrex::Initialize` and `amrex::Finalize` inside `{}` so that things like `PlotFileData` are destroyed before `amrex::Finalize`.

Reply:
done